### ISSUE TITLE
Default graphs in v10.2 RIAB docker not aligned with setup-arcos.sh

### ISSUE THREAD

**Issue by GregIthaca** [ge-high-assurance/RACK#703]
The Rack-in-a-Box docker image for v10.2 appears to come preloaded with two data graphs in the connection, http://rack001/model and http://rack001/nist-800-53. However, the cli/setup-arcos.sh script loads all of the CWE data into the http://rack001/mitre-cwe datagraph, and someone querying from the GUI would likely be led to assume that the loading of the data had failed.

Since the process for adding data graphs is sort of "expert level" knowledge, I think it would work a lot better if this graph were pre-loaded. Presumably if it doesn't contain any data, this would have no further impact on users.

**Comment by kityansiu** [comment]
@GregIthaca, can you please just clarify what you mean by the preloaded data graphs? How are you viewing that there are those two connections loaded? Thanks!

**Comment by GregIthaca** [comment]
In a docker RIAB that has been freshly started and then run through the setup-owl.sh, setup-rack.sh, and setup-arcos.sh, when you connect to the web interface, you can navigate to "Connection" menu and "Load" and then see the default RACK connection. Under this connection, there exists one "Model" graph and two "Data" graphs, but the setup-arcos.sh script has loaded all of the CWE data into a different data graph that is not part of this initial configuration.

Hopefully that's more clear?

**Comment by glguy** [comment]
I don't think we pre-populate the connection settings it's been so long since I had to do anything to change them that I don't remember how that works in SparqlGraph.

It might be the case that the reason your connection string is set the way it is is that a long time ago that's the way you set it up following the getting started instructions from the wiki.

As far as I know we don't distribute any graph URIs by default. Kit or Paul probably know better

**Comment by cuddihyge** [comment]
Yes.  For better or worse, connections are stored in your browser's cookies.
It sounds to me like a reasonable feature request to at least improve documentation and at best build the functionality to pre-load them.   Even though this sounds simple enough, given the current architecture, I don't think there is an obvious/quick solution.  
The easiest partial fix is to preload a simple nodegroup into the store for each connection.  The default connections store along with the nodegroups.   Perhaps this will be enough (?)

**Comment by kityansiu** [comment]
One idea to load the datagraphs for non-expert users is to create a nodegroup with no other purpose but to load the connections. Attached and pictured below is one such example. Note how it purposely has no values selected to return.
![image](https://user-images.githubusercontent.com/12802455/170302378-66ce7b6a-1a30-4220-b7b4-0de0aa4f3595.png)
![image](https://user-images.githubusercontent.com/12802455/170302662-21a17c08-4747-4f57-9174-c4b65cff0e1a.png)
[setup-arcos-ApachePhase2.zip](https://github.com/ge-high-assurance/RACK/files/8772534/setup-arcos-ApachePhase2.zip)

This nodegroup can be loaded via setup-arcos.sh. This can go out as part of v11. In the meantime, this nodegroup can be added to Load-ApacheData.sh in the 10.2 dataset.

**Comment by GregIthaca** [comment]
OK, good to know how that's happening at least. I guess the confusing part here is effectively that the nodegroups for the MITRE CWE data are getting loaded, but don't in themselves cause the additional datagraph to be added to the connection? If so then you probably don't even need a new nodegroup, you just need the existing ingestion queries to create the right graph?

I'm OK with closing this whenever you feel you've captured the actions you wish to take.

**Comment by kityansiu** [comment]
See commit [02d82e4](https://github.com/ge-high-assurance/RACK/commit/02d82e42cce92e2c88d2034f866e3835cc8451ab).
